### DISCUSSION TITLE
Add more default skin that users can choose from

### DISCUSSION THREAD

**Discussion by caulmseh** [ppy/osu#14623] [Ideas]
I stumbled on peppy's tweet here https://twitter.com/ppy/status/1434010230770204679 and I want to remove the stigma that surrounds the default skin. I suggest that the default skins should be more than one kind like how we see and use other community-made skins that increase the legibility of the circles and others for aesthetic and switched-up hitsounds

I also suggest updating the skin list to be less cluttered by supporting multiple skins in one file then have selecting modes at the side like (DT, EZ, and NM.) I think that this will be a welcome change for everyone and can also be a feature on the new, upcoming osu! Partnering with the community through submissions and votes can also speed things up if you planned to implement this idea.

**Comment by rednir** [comment]
>supporting multiple skins in one file then have selecting modes at the side like (DT, EZ, and NM.)

Usually these DT/EZ/NM skins have only a couple file changes and are otherwise exactly the same skin, so it would probably be better if the skin creator could label some skin elements as "DT" or "EZ" and players could switch to them easily through a menu.
